squad/ci/backend/lava.py

```diff
@@ -191,23 +191,37 @@ def __download_full_log__(self, job_id):
         return response.content
 
     def __get_job_logs__(self, job_id):
+        logger.debug("Retrieving logs job: %s" % job_id)
         log_data = self.__download_full_log__(job_id)
-        log_data_yaml = yaml.load(log_data, Loader=yaml.CLoader)
         returned_log = ""
-        for log_entry in log_data_yaml:
-            if log_entry['lvl'] == 'target':
-                if isinstance(log_entry['msg'], bytes):
```

Comment:
I am missing this bit from the new code ...

Reply:
I will add it.